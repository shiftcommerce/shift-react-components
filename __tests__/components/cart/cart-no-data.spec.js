// Libraries
import React from 'react'

// Components
import CartNoData from '../../../src/components/cart/cart-no-data'

test('renders exception message when component is loaded', () => {
  // act
  const wrapper = shallow(
    <CartNoData />
  )

  // assert
  expect(wrapper).toMatchSnapshot()
  expect(wrapper).toIncludeText('You have no items in your cart.')
  expect(wrapper).toIncludeText('If you are expecting items in it, please refresh.')
  expect(wrapper).toIncludeText('If issue still persists, please check with our customer care.')
})
