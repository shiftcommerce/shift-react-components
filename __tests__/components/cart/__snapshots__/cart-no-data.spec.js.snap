// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders exception message when component is loaded 1`] = `
<section
  className="c-cart-no-data"
>
  <p>
    You have no items in your cart. If you are expecting items in it, please refresh.
  </p>
  <p>
    If issue still persists, please check with our customer care.
  </p>
</section>
`;
